### PR TITLE
Feature/curation/june2021 nb

### PR DESCRIPTION
**Pull request by ngbush**
Zooma mappings for terms from single cell experiments 09 June 2021.

### CODE REVIEW

File: zoomage_report.CURATED.tsv [hunk 1]
```diff
@@ -56187,3 +56187,215 @@ strain	mixed C57BL/6J and SV129	http://www.ebi.ac.uk/efo/EFO_0000601	E-TABM-426
...
+cause of death	known or suspected suicide	http://www.ebi.ac.uk/efo/EFO_0007624	E-HCAD-1	HCATisStabAug177276394_S1_L001-GATCGTAGTACAGACG	suicide	mismatch
+cell line	F123	http://www.ebi.ac.uk/efo/EFO_0009319	E-MTAB-6362	SAMEA104470790	F123-CASTx129	mismatch
+cell subtype	TCM	http://purl.obolibrary.org/obo/CL_0000813	E-GEOD-106540	SRR6259614	memory T cell	mismatch
+cell type	megagametophyte	http://purl.obolibrary.org/obo/PO_0025279	E-ENAD-50	SRR7812291	megagametophyte	MATCHES_ONTOLOGY_LABEL
+cell type	neural crest-derived cell	http://purl.obolibrary.org/obo/CL_0011012	E-MTAB-5553	ERR1875223	neural crest cell	mismatch
+cell type	phagocyte	http://purl.obolibrary.org/obo/CL_0000234	E-HCAD-21	SRR10009461-GCACATAAGCGTGAGT	phagocyte	mismatch
+development stage	inflorescence initiation stage	http://purl.obolibrary.org/obo/PO_0021004	E-ENAD-51	SRR12276805-TCGAGGCGTGAGTGAC	inflorescence initiation stage	mismatch
+developmental stage	embryonic stage 6	http://www.ebi.ac.uk/efo/EFO_0005865	E-CURD-21	SAMN06344993-ATTCACACCAAC	embryonic stage 6	mismatch
+developmental stage	human adult stage	http://www.ebi.ac.uk/efo/EFO_0001272	E-HCAD-35	SRR9123049-GACTGCGGTACAAGTA	adult	mismatch
```
Comment (anjaf) [inline zoomage_report.CURATED.tsv:56204]: Maybe this is a little too broad mapping. "adult" doesn't necessarily mean human.

File: zoomage_report.CURATED.tsv [hunk 2]
```diff
@@ -56187,3 +56187,215 @@ strain	mixed C57BL/6J and SV129	http://www.ebi.ac.uk/efo/EFO_0000601	E-TABM-426
...
+inferred cell type - authors labels	CD16+ monocyte	http://purl.obolibrary.org/obo/CL_0002397	E-HCAD-4	group26-GGGATGACAGGTTTCA	CD14-positive, CD16-positive monocyte|primary CD14+/CD16+ monocyte	mismatch
+inferred cell type - authors labels	CD16-negative natural killer cell (NK CD16-)	http://purl.obolibrary.org/obo/CL_0000938	E-MTAB-6701	ERR2743654-CGTTGGGGTATATCCG	CD16-negative, CD56-bright natural killer cell, human	mismatch
+inferred cell type - authors labels	CD16-positive natural killer cell (NK CD16+)	http://purl.obolibrary.org/obo/CL_0000939	E-MTAB-6701	ERR2743646-CTCGGGAGTCTCACCT	CD16-positive, CD56-dim natural killer cell, human	mismatch
+inferred cell type - authors labels	CD19+CD20+ B cell	http://purl.obolibrary.org/obo/CL_0001201	E-MTAB-8410	SAMEA6057392-TCAGGATCACATGACT	B cell, CD19-positive	mismatch
+inferred cell type - authors labels	CD34+ progenitor	http://purl.obolibrary.org/obo/CL_0001059	E-HCAD-1	HCATisStab7587209_S1_L001-GGGCACTCAATCCGAT	common myeloid progenitor, CD34-positive	mismatch
```
Comment (anjaf) [inline zoomage_report.CURATED.tsv:56221]: I would be careful with this one. They probably meant the blood cells here but that's not in the label and other progenitor cell types also express CD34 (I know it from the skin stem cells I've worked with) so this is not a strict derivation from CD34+ progenitor to myeloid progenitor.

File: zoomage_report.CURATED.tsv [hunk 3]
```diff
@@ -56187,3 +56187,215 @@ strain	mixed C57BL/6J and SV129	http://www.ebi.ac.uk/efo/EFO_0000601	E-TABM-426
...
+inferred cell type - authors labels	embryo sac central cell	http://purl.obolibrary.org/obo/PO_0020090	E-ENAD-50	SRR7812263	embryo sac central cell	mismatch
+inferred cell type - authors labels	embryo sac egg cell	http://purl.obolibrary.org/obo/PO_0025123	E-ENAD-50	SRR7812287	embryo sac egg cell	mismatch
+inferred cell type - authors labels	epidermal initial cell	http://purl.obolibrary.org/obo/PO_0000349	E-GEOD-141730	SAMN13520734-GCAATCACAATTCCTT	epidermal initial cell	mismatch
+inferred cell type - authors labels	granulocyte progenitor	http://purl.obolibrary.org/obo/CL_0000557	E-MTAB-9067	ERR4148034	granulocyte monocyte progenitor cell	mismatch
+inferred cell type - authors labels	juxtaglomerular complex cell	http://purl.obolibrary.org/obo/CL_1000618	E-GEOD-124472	SRR8377704-TTCGAAGTCATCGCTC	juxtaglomerular complex cell	mismatch
+inferred cell type - authors labels	lateral root primordia	http://purl.obolibrary.org/obo/PO_0000016	E-GEOD-158761	SAMN16293294-ATGTGTGGTTTACTCT	lateral root primordium	mismatch
+inferred cell type - authors labels	lymph node endothelial cell	http://purl.obolibrary.org/obo/CL_0002138	E-EHCA-2	22028_2_203	endothelial cell of lymphatic vessel	mismatch
```
Comment (anjaf) [inline zoomage_report.CURATED.tsv:56237]: Lymph node isn't quite the same as lymphatic vessel. So probably wouldn't map that.

File: zoomage_report.CURATED.tsv [hunk 4]
```diff
@@ -56187,3 +56187,215 @@ strain	mixed C57BL/6J and SV129	http://www.ebi.ac.uk/efo/EFO_0000601	E-TABM-426
...
+inferred cell type - authors labels	pDC	http://purl.obolibrary.org/obo/CL_0000784	E-CURD-46	SAMN12356588-GAGCAGAAGGACAGAA	plasmacytoid dendritic cell	mismatch
+inferred cell type - authors labels	periportal liver sinusoidal endothelial cell	http://www.ebi.ac.uk/efo/EFO_0010705	E-HCAD-9	group4-TCTCATATCGCCTGTT	periportal hepatic sinusoidal endothelial cell	mismatch
+inferred cell type - authors labels	phagocyte	http://purl.obolibrary.org/obo/CL_0000234	E-HCAD-35	SRR9123042-GGTGCGTTCTAGAGTC	phagocyte	mismatch
+inferred cell type - authors labels	phloem pole pericycle	http://purl.obolibrary.org/obo/PO_0000065	E-GEOD-158761	SAMN16293297-TTTGGTTGTCCGACGT	phloem pole pericycle cell	mismatch
+inferred cell type - authors labels	pith	http://purl.obolibrary.org/obo/PO_0006109	E-ENAD-51	SRR12276805-CTAACTTGTCTCTTAT	pith	mismatch
+inferred cell type - authors labels	placental extravillous trophoblast cell (EVT)	http://purl.obolibrary.org/obo/CL_0008036	E-MTAB-6701	ERR2743662-GGACAAGAGTACGTAA	extravillous trophoblast	mismatch
+inferred cell type - authors labels	PSC cell	http://purl.obolibrary.org/obo/CL_0002410	E-MTAB-5061	ERR1630080	pancreatic stellate cell	mismatch
```
Comment (anjaf) [inline zoomage_report.CURATED.tsv:56253]: PSC could also stand for "pluripotent stem cell"

File: zoomage_report.CURATED.tsv [hunk 5]
```diff
@@ -56187,3 +56187,215 @@ strain	mixed C57BL/6J and SV129	http://www.ebi.ac.uk/efo/EFO_0000601	E-TABM-426
...
+inferred cell type - ontology labels	ionocyte	http://purl.obolibrary.org/obo/CL_0005006	E-CURD-52	SRR11589514-TATCATAGCGAG	ionocyte	
+inferred cell type - ontology labels	lamellocyte	http://purl.obolibrary.org/obo/CL_0000396	E-CURD-56	SRR11263267-GGGAATTCGGAC	lamellocyte	
+inferred cell type - ontology labels	lamellocyte	http://purl.obolibrary.org/obo/CL_0000396	E-GEOD-141273	SRR10574222-CTGAGTCAGCCA	lamellocyte	
+inferred cell type - ontology labels	lamellocyte	http://purl.obolibrary.org/obo/CL_0000396	E-MTAB-8698	ERR3833766-TCACGAATCCTTGCCA	lamellocyte	
+inferred cell type - ontology labels	preosteoblast	http://purl.obolibrary.org/obo/CL_0007010	E-CURD-12	SAMN11526604-ACAGCTAAGACTGGGT	preosteoblast	
+inferred cell type - ontology labels	trichoblast	http://purl.obolibrary.org/obo/PO_0000262	E-GEOD-141730	SAMN13520735-CGGACTGAGATCCTGT	trichoblast	
+facs marker	CD141+	http://purl.obolibrary.org/obo/CL_0002394	E-GEOD-89232	SAMN05946161	CD141-positive myeloid dendritic cell	mismatch
+facs marker	CD1c+	http://purl.obolibrary.org/obo/CL_0002399	E-GEOD-89232	SAMN05946938	CD1c-positive myeloid dendritic cell	mismatch
```
Comment (anjaf) [inline zoomage_report.CURATED.tsv]: Again, I think I would leave out any of these rather unspecific surface marker annotations. They could potentially map to other cell types too. Same for the "CD138 positive" immunophenotype mapping below.

**Review by anjaf — APPROVED** [review]
Very good, well done and thanks for your work @ngbush 

A few comments about some of the mappings left in-line.

File: zoomage_report.CURATED.tsv (hunk 4 above)
Comment (ngbush) [reply zoomage_report.CURATED.tsv:56253]: OK so with the PSC cell - I also thought pluripotent stem cell but it comes from a pancreas data set and looking at the submission (https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5061/) I felt it was more likely to be pancreas stellate cell?

Comment (anjaf) [reply zoomage_report.CURATED.tsv:56253]: Yes, you're right. It is that in this particular case. But the mapping is general and doesn't know about this context. If you left that in, next time someone annotates "PSC cell" and means pluripotent stem cell, it automatically gets (wrongly) mapped to stellate cell. Not that we should annotate with acronyms any how ;) ... but to illustrate the idea.

Comment (ngbush) [reply zoomage_report.CURATED.tsv:56253]: Right! OK I see. I will leave it out :)

**Pull request merged**